### ISSUE TITLE
Init model for training

### ISSUE THREAD

**Issue by codaibk** [Engineering-Course/LIP_SSL#27]
Hi, is the init model for training taken from DeepLab-Attention-COCO (`http://liangchiehchen.com/projects/DeepLab_Attention_COCO.html`) ?
Thank

**Comment by Engineering-Course** [comment]
Yes.

**Comment by codaibk** [comment]
Hi, How about with nossl? whats the init model i need to use? my sketch-segment data returns bad result (just one image with one color) after training when i use attention init model for training.

**Comment by Engineering-Course** [comment]
You can also use the model pre-trained on COCO for no SSL. But the models are pre-trained on RGB images and your data are sketch images. It may be one of the reasons for worse results.

**Comment by lyxlynn** [comment]
I also have questions about init models.
You mentioned two training steps in your paper: "First , we train the basic network on our LIP dataset for 30epochs,which takes aboout two days.Then  we perform 'self-supervised' stratage to fine-tune out model with structure-sensitive loss.
1, which step is the  config/attention/solver.prototxt you provided  for?  
2, the fist step to the basic network needs pretrained model  from http://liangchiehchen.com/projects/DeepLab_Attention_COCO.html?  Can I use no-sll as the basic network?
3.If I want to train no-sll, can I just train from  model pre-trained on COCO ?

**Comment by Engineering-Course** [comment]
1. The second step
2 & 3. Yes, you can
